source/Octopus.Tentacle/Services/Capabilities/CapabilitiesService.cs

```diff
@@ -9,7 +9,7 @@ public class CapabilitiesService : ICapabilitiesService
     {
         public CapabilitiesResponse GetCapabilities()
         {
-            return new CapabilitiesResponse(new List<string>());
+            return new CapabilitiesResponse(new []{ nameof(IScriptServiceV2)+"Alpha" });
```

Comment:
This needs fixing in another PR but there is an issue in server using the capability service due to deserialization